scripts/RESTORE.md

```diff
@@ -0,0 +1,82 @@
+# Backup and Restore Guide
+
+## Backup Process
+
+### Creating a Backup
+
+1. **Ensure services are running** - The backup process requires the database to be accessible
+2. **Run the backup script**:
+   ```bash
+   ./scripts/backup.sh
+   ```
+3. **Backup location** - Backups are stored in `data/backups/` directory
+4. **Backup naming** - Files are named with timestamp: `shutter-api-keyper-YYYY-MM-DDTHH-MM-SS.tar.xz`
+
+### What Gets Backed Up
+
+- Database dump (`keyper.dump`) - Contains full schema and data from the `keyper` database
+- Chain data (`data/chain/`) - Blockchain data and configuration
+- Keyper configuration (`config/`) - Application configuration files
+- Environment variables - Metrics configuration settings
+
+## Restore Process
+
+### Prerequisites
+
+- **Empty keyper instance** - The restore should be performed on a fresh, empty deployment
+- **No running services** - Ensure all Docker containers are stopped before restore
+- **Backup file available** - The backup archive should be present in `data/backups/` directory
+
+### Restore Steps
+
+1. **Setup environment**:
+   ```bash
+   cp example-api.env .env
```

Comment:
See below, I think we should backup and restore the whole env file (excluding the private key possibly).